app/src/main/java/com/scouts/kitchenplaner/model/usecases/DisplayProjectOverview.kt

```diff
@@ -97,10 +113,20 @@ class DisplayProjectOverview @Inject constructor(
             }
     }
 
+    /**
+     * Archives a project. Only the meta data is present on the device, if it  is a online project
```

Comment:
"Archives a project. If it is an online project, only the meta data will remain on the device."

What happens if it isn't?